tests/foreman/cli/test_remoteexecution.py

```diff
@@ -528,13 +500,30 @@ def test_positive_install_multiple_packages_with_a_job(self):
         """
         packages = ["cow", "dog", "lion"]
         # Create a custom repo
-        setup_org_for_a_custom_repo({
-            u'url': FAKE_0_YUM_REPO,
-            u'organization-id': self.org['id'],
-            u'content-view-id': self.content_view['id'],
-            u'lifecycle-environment-id': self.env['id'],
-            u'activationkey-id': self.activation_key['id'],
-        })
+        repo = entities.Repository(
+                content_type='yum',
```

Comment:
Wrong indentation - should be 4 spaces instead of 8:
```python
repo = entities.Repository(
    content_type='yum',
    product=entities.Product(organization=self.org).create(),
    url=FAKE_0_YUM_REPO,
).create()
```
Comment is applicable for most of indentations in this PR, i don't want to spam with similar comments.

Btw nailgun's repo would even fail travis in case of such non-PEP8 style, we should probably think of adding pylint to robottelo too.

Reply:
will fix, thanks